src/commands/init/generateTemplate/index.js

```diff
@@ -54,7 +53,7 @@ export default function generateTemplate(name, src, dest, done) {
             }
             log.success('Project created');
             done();
-            logMessage(opts.completionMessage, data);
+            logMessage(opts.completionMessage, { ...data, ...metalsmith.metadata() });
```

Comment:
Good point, but I'd lean more towards validating keys in prompts as soon as they come and making sure that `destDirName` and `inPlace` are not used, because otherwise we end up with some confusion ("I've declared my inPlace and it's a different value in completionMessage and in rendered template").
Early error "`inPlace` is a reserved property" may help us avoid it.